console/src/components/Services/EventTrigger/Add/AddTrigger.js

```diff
@@ -517,7 +517,14 @@ class AddTrigger extends Component {
                 }
               >
                 <option value="">Select table</option>
-                {tableListBySchema.map(t => {
+                {tableListBySchema.sort((a, b) => {
```

Comment:
Let's move the sort to the main function to reflect it in other parts of the app where its reused.